packages/starlight/components/PrevNextLinks.astro

```diff
@@ -77,4 +78,12 @@ const t = useTranslations(locale);
 		font-size: var(--sl-text-2xl);
 		line-height: var(--sl-line-height-headings);
 	}
+
+	svg {
+		flex-shrink: 0;
+	}
+
+	svg + span {
+		width: calc(100% - 1.5rem);
+	}
```

Comment:
If I am not missing something, I guess like in the `<SidebarSublist/>` component, this could be avoided by using `overflow-wrap: anywhere;` instead of `overflow-wrap: break-word;` for the link.

Any thoughts?

Reply:
Based on the previous comments and fixes, this has been removed.